internal/runner/client/client_local.go

```diff
@@ -91,9 +108,9 @@ func (r *LocalRunner) newExecutable(task project.Task) (runner.Executable, error
 		Tty:     block.InteractiveLegacy(),
 		Stdout:  r.stdout,
 		Stderr:  r.stderr,
-		Session: r.session,
 		System:  system.Default,
 		Logger:  r.logger,
+		Session: r.session,
```

Comment:
Not sure why this "had" to move down a few lines - not wrong but no change should be no diff?